styx-service-common/src/test/java/com/spotify/styx/api/AuthenticatorTest.java

```diff
@@ -20,6 +20,8 @@
 
```

Comment:
Is there no test we need to change for the 404->403 code change in styx-service-common/src/main/java/com/spotify/styx/api/Authenticator.java ?

Reply:
https://github.com/spotify/styx/pull/1092/files#diff-6d62dc7647527de82216a154436bab32cb014d6a27fc005028168031ea1a95f2R313-R332f those are the changes. There is no externally observable difference though because we treat exceptions the same except the logging.

Reply:
ah true